modules/cert-manager-enable-operand-log-level.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-log-levels.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-enable-operand-log-level_{context}"]
+= Setting a log level for cert-manager
+
+You can set a log level for cert-manager to determine the amount of detail in logged error messages.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* The {cert-manager-operator} is installed.
+
+.Procedure
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager.operator cluster
+----
+
+. Set the log level value by editing the `spec.logLevel` section:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  creationTimestamp: "2023-05-10T07:34:15Z"
+  finalizers:
+  - cert-manager-operator.operator.openshift.io/cert-manager-controller-deployment
+  - cert-manager-operator.operator.openshift.io/cert-manager-webhook-deployment
+  - cert-manager-operator.operator.openshift.io/cert-manager-cainjector-deployment
+  generation: 1
+  name: cluster
+  resourceVersion: "47290"
+  uid: 0f5cd145-a20f-43d1-ac3d-0db28a6f9f06
+spec:
+  logLevel: Normal <1>
+----
+<1> Replace `<log_level>` with the desired log level value. The default `logLevel` is `Normal`. The valid log level values for the `CertManager` resource are `Normal`, `Debug`, `Trace`, and `TraceAll`. You can set `logLevel` to `Normal` to audit logs and perform common operations when everything is fine. To troubleshoot a minor issue, you can set `logLevel` to `Debug` to view verbose logs. To troubleshoot a major issue, you can set `logLevel` to `Trace` to view more verbose logs. To troubleshoot serious issues, you can set `logLevel` to `TraceAll`.
++
+[NOTE]
+====
+`TraceAll` generates huge amount of logs. After setting `logLevel` to `TraceAll`, you might experience performance issues.
+====
+
+. Save the file to apply the changes.
```

Comment:
It does oc edit to save changes for a resource, not usual sense of save a `file` (though oc edit uses temporary file)
```suggestion
. Save your changes and quit the text editor to apply your changes.
```